sync/class.jetpack-sync-module-themes.php

```diff
@@ -25,6 +28,48 @@ public function init_listeners( $callable ) {
 		add_action( 'jetpack_widget_reordered', $callable );
 	}
 
+	public function sync_network_allowed_themes_change( $option, $value, $old_value, $network_id ) {
+		if ( count( $old_value ) > count( $value ) )  {
+			$disabled_theme_names = array_keys( array_diff_key( $old_value, $value ) );
+			$disabled_themes = $this->get_theme_details_for_slugs( $disabled_theme_names );
+			/**
+			 * Trigger action to alert $callable sync listener that network themes were disabled
+			 *
+			 * @since 5.0.0
+			 *
+			 * @param mixed $disabled_themes, Array of info about network disabled themes
+			 * @param mixed array_keys( $value ), Array of slugs of all enabled themes
+			 */
+			do_action( 'jetpack_network_disabled_themes', $disabled_themes, array_keys( $value ) );
+			return;
+		}
+
+		$enabled_theme_names = array_keys( array_diff_key( $value, $old_value ) );
+		$enabled_themes = $this->get_theme_details_for_slugs( $enabled_theme_names );
+		/**
+		 * Trigger action to alert $callable sync listener that network themes were enabled
+		 *
+		 * @since 5.0.0
+		 *
+		 * @param mixed $enabled_themes , Array of info about network enabled themes
+		 * @param mixed array_keys( $value ), Array of slugs of all enabled themes
+		 */
+		do_action( 'jetpack_network_enabled_themes', $enabled_themes, array_keys( $value ) );
```

Comment:
Can we make array_keys( $value ) more descriptive. Something like 
`$enabled_theme_slugs` would read better